src/Main.java

```diff
@@ -1,5 +1,110 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        System.out.println("Задание 1");
+        byte cat = 5;
+        System.out.println(cat);
+        short dog = 300;
+        System.out.println(dog);
+        int pig = 60000;
+        System.out.println(pig);
+        long starFish = - 1234567891;
+        System.out.println(starFish);
+        float squid = 5.3f;
+        System.out.println(squid);
+        double friedFish = 1.8f;
+        System.out.println(friedFish);
+
+        System.out.println("Задание 2");
+        double a = 27.12f;
+        System.out.println(a);
+        long b = 987678965549L;
+        System.out.println(b);
+        double c = 2.786f;
+        System.out.println(c);
+        short d = 569;
+        System.out.println(d);
+        short e = - 159;
+        System.out.println(e);
+        int f = 27897;
+        System.out.println(f);
+        byte g = 67;
+        System.out.println(g);
+
+        System.out.println("Задание 3");
+        byte LyudmilaPavlovna = 23;
+        byte AnnaSergeevna = 27;
+        byte EkaterinaAndreevna = 30;
+        int sheets = 480;
+        int students = LyudmilaPavlovna + AnnaSergeevna + EkaterinaAndreevna;
+        int sheetsPerStudent = sheets / students;
+        System.out.println("На каждого ученика рассчитано " + sheetsPerStudent + " листов бумаги");
+
+        System.out.println("Задание 4");
+        byte bottlesPer2Minutes = 16;
+        byte minutesInHour = 60;
+        byte hoursInDay = 24;
+        byte daysInMonth = 30;
+        int bottlesPer20Minutes = (bottlesPer2Minutes / 2) * 20;
+        int bottlesPerDay = (bottlesPer2Minutes / 2) * minutesInHour * hoursInDay;
+        int bottlesPer3Days = (bottlesPer2Minutes / 2) * minutesInHour * hoursInDay * 3;
+        int bottlesPerMonth = (bottlesPer2Minutes / 2) * minutesInHour * hoursInDay * daysInMonth;
```

Comment:
```suggestion
        int bottlesPerMonth = 10 * bottlesPer3Days;
```
используем уже посчитанные результаты, а не считаем заново